docs/en/guides/choosing-a-test-runner.md

```diff
@@ -1,45 +1,45 @@
-# Choosing a test runner
+# Choisir un lanceur de tests
 
-A test runner is a program that runs tests.
+Un lanceur de tests est un programme qui lance des tests (et oui !).
 
-There are many popular JavaScript test runners, and `vue-test-utils` works with all of them. It's test runner agnostic.
+Il y a un nombre conséquent de lanceurs de tests JavaScript et `vue-test-utils` marche avec tous ceux d'entre eux. `vue-test-utils` est agnostique de lanceurs de tests. 
 
-There are a few things to consider when choosing a test runner though: feature set, performance, and support for single-file component (SFC) pre-compilation. After carefully comparing existing libraries, here are two test runners that we recommend:
+Il y a tout de même plusieurs choses à considérer afin de choisir un lanceur de tests : variété des fonctionnalités, performance et support de la pré-compilation des composants monofichiers (SFC). Après avoir cautionnement analysé et comparé les bibliothèques, il y a deux lanceurs de tests que nous recommandons :
```

Comment:
(« SFC »)

Je suis d'avis qu'il est tellement pas populaire se terme que je sais pas s'il y a un intéret de le préciser (en général on précise ça si on traduit un terme populaire en anglais genre « end-to-end » ou « bundle » ou « SPA » mais SFC... inconnu au bataillon :D